witness/golang/omniwitness/.env

```diff
@@ -0,0 +1,11 @@
+WITNESS_PRIVATE_KEY=PRIVATE+KEY+YourTokenHere+2fed0a1a+Aaw5ZWCAKxLPYYiZybLJecpf4mR4hRxIWf8SQlXeTVkg
+WITNESS_PUBLIC_KEY=YourTokenHere+2fed0a1a+AcVxZHWbfeJepQ7Ltv5G4kvJfNo8hacOdcy1JGZ85E79
+
+GITHUB_AUTH_TOKEN=ghp_XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+GIT_USERNAME=johndoe
+GIT_EMAIL=johndoe@example.com
+
+MHUTCHINSON_DISTRIBUTOR_FORK=johndoe/mhutchinson-distributor
+WOLSEY_BANK_FORK=johndoe/rediffusion
```

Comment:
Hmm, can `.env` files do shell expansion (e.g. `MHUTCH_DISTRIB_FORK=${GIT_USERNAME}/mhutchinson-distributor`)?

Reply:
I just inlined these in the end. This type of expansion doesn't work.